ssl/quic/quic_impl.c

```diff
@@ -1102,45 +1351,63 @@ int ossl_quic_get_event_timeout(SSL *s, struct timeval *tv, int *is_infinite)
          */
         tv->tv_sec  = 1000000;
         tv->tv_usec = 0;
-
-        quic_unlock(ctx.qc);
         return 1;
     }
 
-    *tv = ossl_time_to_timeval(ossl_time_subtract(deadline, get_time(ctx.qc)));
+    /*
```

Comment:
I think this logic here should be folded to get_time() itself. (the get_time() is local function to `quic_impl.c`
Perhaps I would just revert those changes from rebase. If i remember correct all regression tests did pass without this change being present.